pyproject.toml

```diff
@@ -93,7 +93,7 @@ mammos-mumag = { path = ".", editable = true }
 [tool.pixi.tasks]
 examples = "jupyter-lab examples"
 test-unittest = "pytest -v tests"
-test-docstrings = "pytest -v --doctest-modules src/mammos_mumag"
+test-docstrings = "run-escript -- -m pytest -v --doctest-modules src/mammos_mumag"
```

Comment:
Why do we need `run-escript` for the doctests but not unittests and notebook tests?